parmed/openmm/parameters.py

```diff
@@ -286,6 +287,23 @@ def nowild(name):
             )
         dest.write(' </CustomTorsionForce>\n')
 
+    def _write_omm_urey_bradley(self, dest):
+        if not self.urey_bradley_types: return None
+        dest.write(' <!-- Urey-Bradley terms -->\n')
+        dest.write(' <AmoebaUreyBradleyForce>\n')
+        length_conv = u.angstroms.conversion_factor_to(u.nanometers)
+        _ambfrc = u.kilocalorie_per_mole/u.angstrom**2
+        _ommfrc = u.kilojoule_per_mole/u.nanometer**2
+        frc_conv = _ambfrc.conversion_factor_to(_ommfrc)
+        ureys_done = set()
+        for (a1, a2, a3), urey in iteritems(self.urey_bradley_types):
+            if (a1, a2, a3) in ureys_done: continue
+            if urey == NoUreyBradley: continue
+            dest.write('  <UreyBradley type1="%s" type2="%s" type3="%s" d="%f" k="%f"/>\n'
+                       % (a1, a2, a3, urey.req*length_conv, urey.k*frc_conv))
```

Comment:
Is `frc_conv` missing a factor of 2 here?  ParmEd stores bond type force constants according to the functional form `k(r-req)^2`, whereas OpenMM in most places uses `1/2*k*(r-req)^2`